samples/tfm/provisioning_image/sample.yaml

```diff
@@ -0,0 +1,18 @@
+sample:
+  description: Initiates the provisioning of a TF-M image.
+  name: Provisioning Image
+common:
+    tags: keys
+    platform_allow: nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp
+      - nrf9160dk_nrf9160
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "Writing the identity key to KMU"
+        - "Success!"
+tests:
+  sample.keys.identity_key_generate.random_key:
+    tags: keys ci_build
```

Comment:
@torsteingrindvik are these tag sufficient enough to run this sample in the tests or are there test changes needed or other tags here?